sse2neon.h

```diff
@@ -8500,12 +8500,8 @@ FORCE_INLINE uint32_t _mm_crc32_u8(uint32_t crc, uint8_t v)
     crc = __crc32cb(crc, v);
 #else
     crc ^= v;
-    for (int bit = 0; bit < 8; bit++) {
-        if (crc & 1)
-            crc = (crc >> 1) ^ UINT32_C(0x82f63b78);
-        else
-            crc = (crc >> 1);
-    }
+    for (int bit = 0; bit < 8; bit++)
+        crc = (crc & 1) ? ((crc >> 1) ^ UINT32_C(0x82f63b78)) : (crc >> 1);
```

Comment:
Don't do this. The optimizing compilers can generate the identical machine code, and the original code is easier to maintain.

Reply:
Revert completed.